client/handler.go

```diff
@@ -21,10 +21,14 @@
 package client
 
 import (
+	"context"
 	"encoding/json"
+	"io"
 	"net/http"
 	"time"
 
+	jsonpatch "github.com/evanphx/json-patch"
```

Comment:
Is this the best library for the job? What alternatives do we have?

Reply:
The [JSON Patch](http://jsonpatch.com/) site gives two options for Go implementations, this and another library, https://github.com/mattbaird/jsonpatch. Between the two, this one has more starts (551 vs 89) and more contributors (41 vs 8). This one was also updated more recently (January vs last August). Additionally, this library also supports [Merge Patch](https://tools.ietf.org/html/rfc7396), which was indicated as a potential alternative in https://github.com/ory/meta/issues/2. 

For these reasons, I felt this was the better choice between the two.

Reply:
Perfect, thank you!